### DISCUSSION TITLE
Text inside search in PillsInput component is not selectable via mouse

### DISCUSSION THREAD

**Discussion by OleksiiSamutin** [mantinedev/mantine#5317] [Q&A] [answered]
The text inside PillsInput.Field search is not selectable, and I don't understand why in TextInput component it works as intented, and I can select text which I type in Search, but in PillsInput it doesn't work (If look in example page for PillsInput text also is not selectable). Is there any way to make PillsInput.Field search text to be selectable?

**Comment by rtivital** [comment, marked as answer]
It seems like a bug, I’ll try to fix it in one of the next patches.
